cpp/conv_templates.cc

```diff
@@ -54,6 +54,40 @@ Conversation Llama2() {
   return conv;
 }
 
+Conversation CodeLlamaCompletion() {
```

Comment:
Can we add this to web-llm's implementation.
https://github.com/mlc-ai/web-llm/blob/main/src/conversation.ts#L164

Why is this in two places? Shouldn't we be able to add conversation templates without having to update the code base and then re-releasing?